net-im/nheko/nheko-0.12.1.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake optfeature xdg
+
+MY_P="${PN}-v${PV}"
+DESCRIPTION="Native desktop client for Matrix using Qt"
+HOMEPAGE="https://github.com/Nheko-Reborn/nheko"
+SRC_URI="https://nheko.im/nheko-reborn/${PN}/-/archive/v${PV}/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-3 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="X doc +pie man video voip wayland"
+REQUIRED_USE="video? ( voip )"
+
+# Check WebRTCSession::havePlugins() in ./src/voip/WebRTCSession.cpp
+# to get GStreamer plugin deps right.
+MY_GST_V="1.20.0"
+
+QTMIN="6.5.0"
+COMMON_DEPEND="
+	app-text/cmark:=
+	>=dev-cpp/blurhash-0.2.0:=
+	dev-cpp/cpp-httplib:=
+	dev-db/lmdb:=
+	dev-libs/libfmt:=
+	>=dev-libs/kdsingleapplication-1.1.0
+	>=dev-libs/mtxclient-0.10.1:=
+	dev-libs/olm
+	>=dev-libs/qtkeychain-0.14.1-r1:=
```

Comment:
A better way to write this dependency is `dev-libs/qtkeychain:=[qt6(+)]`

Reply:
Will change it. Also, I think that gst-plugins- packages can be unrestricted as the version on the tree is already higher than what is required